docs/UserGuide.md

```diff
@@ -548,7 +548,7 @@ find KEYWORD/ID...
 
 | Command           | Reason                                                                                              |
 |-------------------|-----------------------------------------------------------------------------------------------------|
```

Comment:
I realised that find shouldnt work on applications so we shld remove that part

Reply:
Ok have removed it